script/tool/lib/src/version_check_command.dart

```diff
@@ -655,4 +690,55 @@ ${indentation}The first version listed in CHANGELOG.md is $fromChangeLog.
 
     return null;
   }
+
+  Future<bool> _validateBatchRelease({
```

Comment:
This should get a comment explaining when it should be called; it's non-obvious that it must only be called if both the package is using batched releases *and* this isn't a merge-back PR.